dev/tasks/conda-recipes/variants/linux_python2.7.yaml

```diff
@@ -1,13 +1,11 @@
 boost_cpp:
```

Comment:
These variant files are a bit outdated. They should at least include the `libprotobuf: 3.7` bump to be inline with the conda-forge dependencies.

Reply:
Right, updating it.

Reply:
Updated the protobuf pin (will update the rest after a smithy rerender on the feedstocks):
[kszucs/crossbow/build-538](https://github.com/kszucs/crossbow/branches/all?utf8=%E2%9C%93&query=build-538)

Reply:
out of curiousity what's the procedure for maintaining these files inside apache/arrow? can we run conda-smithy rerender locally? The files are getting more complicated

Reply:
We cannot, smithy is tightly coupled with the repository structure of a feedstock. This is a manual procedure :(

Reply:
I'm reverting the pin, because orc feedstock pins libprotobuf < 3.7 causing dependency resolution error.

Reply:
Fixed the package https://github.com/conda-forge/orc-feedstock/pull/3

Reply:
With the pin, conda can resolve but will use old packages for dependencies.